app/_locales/en/messages.json

```diff
@@ -1950,7 +1950,7 @@
     "message": "Below are all the quotes gathered from multiple liquidity sources."
   },
   "swapSlippageTooLow": {
-    "message": "Slippage must be greater than zero"
+    "message": "Slippage must be greater or equal to zero"
```

Comment:
I believe it's fine if only this English translation change is pushed to production first, it can be translated to other languages in another release. It would be great if our localization team can update values for the existing key (`swapSlippageTooLow`) for other languages, otherwise I can create a new key as well for this content change.

Reply:
So we should rename the key of this locale to reflect the new meeting, and to make clear that new translations are needed.

Reply:
Sounds good, I will rename the key in this `en` file. What about `swapSlippageNegative`? And then in our code, should we temporarily use the code below, until it's translated for all languages that we support?
```
errorText = t('swapSlippageNegative') || t('swapSlippageTooLow');
```

Once it's translated for all languages, I would change code to:
```
errorText = t('swapSlippageNegative');
```

Reply:
Update: I only use `errorText = t('swapSlippageNegative');` now and have removed all `swapSlippageTooLow` keys.